<launch>

<!-- start roomba driver node -->
	<node pkg="roomrider" type="roomba_driver" output="screen" name="roomba" >
	</node>

<!-- start laser driver node -->
	<node pkg="roomrider" type="sick300_driver" output="screen" name="sick300" >
	</node>
	
<!-- start joy driver node -->
	<node pkg="joy" type="joy_node" output="screen" name="joystick" >
	</node>
	
<!-- start roomba_test driver node -->
	<node pkg="roomrider" type="roomba_test" output="screen" name="roombaTest" >
	</node>


</launch>